Jenkinsfile

```diff
@@ -463,11 +478,8 @@ stage('Unit Test') {
       } else {
          Utils.markStageSkippedForConditional('python3: arm')
       }
-    }
-}
-
-stage('Integration Test') {
```

Comment:
however I'm not sure it's a good idea to run the frontend tests as non-cancellable without unit tests passing

Reply:
let's give this a shot and we can always bring forward the failFast logic from @mousius now that we are going this route